test/test_sparse.py

```diff
@@ -255,6 +255,22 @@ def test_tensor(x, res):
         res = self.ValueTensor(3, 4, 5, 0)
         test_tensor(x, res)
 
+    @skipIfRocm  # ROCm see https://github.com/pytorch/pytorch/pull/12171#issuecomment-431069849
+    def test_to_sparse(self):
+        shape = [10, 5, 19, 8]
+        for dim, dim_sz in enumerate(shape):
+            sparse_dims = dim + 1
+            for nnz in [0, 1, int(dim_sz / 3)]:
```

Comment:
why nnz needs to be `int(dim_sz / 3)`?

Reply:
I think it can be anything. Should it be random?

Reply:
yes, you can pick a valid randint in this case